.github/workflows/create-release.yml

```diff
@@ -0,0 +1,53 @@
+name: Create release candidate
+
+on:
+  workflow_dispatch:
+    inputs:
+      releaseLevel:
```

Comment:
Since we use conventional commits, I don't think this input is needed. Instead, we should just let `standard-version` do its job.